apps/api/src/domain/models/vocabulary-list/queries/vocabulary-list-queries-fetch-by-id.e2e.spec.ts

```diff
@@ -335,26 +339,383 @@ describe.skip(`when querying for a vocabulary list: fetch by ID`, () => {
+                            });
+
+                            const entriesWithUnpublishedTerm = [
+                                {
+                                    term: publishedTerm,
+                                    variableValues: {},
+                                },
+                                {
+                                    term: unpublishedTerm,
+                                    variableValues: {},
+                                },
+                            ];
+
+                            await app
+                                .get<IVocabularyListQueryRepository>(
+                                    VOCABULARY_LIST_QUERY_REPOSITORY_TOKEN
+                                )
+                                .create(
+                                    publishedVocabularyList.clone({
+                                        entries: entriesWithUnpublishedTerm,
+                                    })
+                                );
+                        },
+                        checkResponseBody: async ({
+                            entries,
+                        }: IDetailQueryResult<IVocabularyListViewModel>) => {
+                            // the unpublished term should come through for an admin user
+                            expect(entries).toHaveLength(2);
+                        },
+                    });
+                });
+            });
+        });
+
+        describe(`when the vocabulary list is not public`, () => {
+            it(`should still return the vocabulary list`, async () => {
+                await assertQueryResult({
+                    app,
+                    endpoint: buildDetailEndpoint(vocabularyListId),
+                    expectedStatus: HttpStatusCode.ok,
+                    seedInitialState: async () => {
+                        await app
+                            .get<IVocabularyListQueryRepository>(
+                                VOCABULARY_LIST_QUERY_REPOSITORY_TOKEN
+                            )
+                            .create(
+                                publishedVocabularyList.clone({
+                                    isPublished: false,
+                                    accessControlList: new AccessControlList().toDTO(),
+                                    entries: [
+                                        {
+                                            term: publishedTerm.clone({
+                                                isPublished: false,
+                                                accessControlList: new AccessControlList().toDTO(),
+                                            }),
+                                            variableValues: {},
+                                        },
+                                    ],
+                                })
+                            );
+                    },
+                    checkResponseBody: async ({
```

Comment:
I haven't needed the async yet, but I suspect we may hit cases where we want to make subsequent requests to compare additional state. Just in case, I made it async up front.